pypmc/sampler/markov_chain.py

```diff
@@ -206,10 +215,10 @@ def __init__(self, *args, **kwargs):
         self.unscaled_sigma = self.proposal.sigma / self.covar_scale_factor
 
     @_inherit_docstring(MarkovChain)
-    def run(self, N=1):
+    def run(self, N=1, continue_on_NaN=False):
```

Comment:
need to update docs as well

Reply:
The method run is documented via 'autoclass'. The added keyword argument is documented in ```MarkovChain``` via docstring, and is reused here.

Reply:
yep, should work